tasks/Reading the mind in the eyes.md

```diff
@@ -0,0 +1,78 @@
+![image](https://user-images.githubusercontent.com/78745728/131226818-56a37662-1a8b-40eb-993a-7af4eba9c152.png)
+![image](https://user-images.githubusercontent.com/78745728/131226823-a222647b-e343-4901-a813-ab89cd1534a2.png)
+![image](https://user-images.githubusercontent.com/78745728/131226828-8dc59292-b22a-4d85-b189-fe2379f3deb1.png)
+![image](https://user-images.githubusercontent.com/78745728/131226834-c14be2ed-e3ff-4c36-837d-e111dff34e46.png)
+![image](https://user-images.githubusercontent.com/78745728/131226839-830f5ac1-de1d-486c-aa6a-21cfb79b3287.png)
+![image](https://user-images.githubusercontent.com/78745728/131226844-5a187c14-4f67-4bb9-85c5-a8e1ce219488.png)
+![image](https://user-images.githubusercontent.com/78745728/131226849-477dd10e-4dea-49ef-bbff-6a41055c482f.png)
+![image](https://user-images.githubusercontent.com/78745728/131226857-0303caa6-1eb3-4a4d-b399-e756aa1f0ba7.png)
+![image](https://user-images.githubusercontent.com/78745728/131226864-7c8ee66a-b5d8-4170-be3d-189e1bd96c4e.png)
+![image](https://user-images.githubusercontent.com/78745728/131226870-3ace180d-7323-40f7-a3a0-fb3782112fed.png)
+![image](https://user-images.githubusercontent.com/78745728/131226874-9b211d15-e681-49c8-97fb-0d8cea03dc41.png)
+![image](https://user-images.githubusercontent.com/78745728/131226880-5d435dec-8296-453c-92bc-96707217c63a.png)
+![image](https://user-images.githubusercontent.com/78745728/131226886-1b066a69-db2f-4990-9109-d92b1a2b3560.png)
+![image](https://user-images.githubusercontent.com/78745728/131226889-ab96811d-6c33-4ddd-9158-20914cef9e3e.png)
+![image](https://user-images.githubusercontent.com/78745728/131226892-63eee908-581a-4173-bb2c-987221f3bb8c.png)
+![image](https://user-images.githubusercontent.com/78745728/131226902-c8d14ef3-ef21-4027-b489-729e59ecc7cd.png)
+![image](https://user-images.githubusercontent.com/78745728/131226905-9dfbfc7b-66ac-4cb0-a292-2691505b774b.png)
+![image](https://user-images.githubusercontent.com/78745728/131226908-cec29cc5-348d-439a-a3fb-a0bde04ccc0f.png)
+![image](https://user-images.githubusercontent.com/78745728/131226913-bd604d63-bd1b-43ab-9add-1ea7b43bffc2.png)
+![image](https://user-images.githubusercontent.com/78745728/131226917-b0e61983-f000-4035-9b59-ab46170fccdc.png)
+![image](https://user-images.githubusercontent.com/78745728/131226920-56e79b7a-ba33-4f29-8f02-6403f30dec16.png)
+![Practice image](https://user-images.githubusercontent.com/78745728/131226925-b68ad856-5bc7-4d3e-ad16-d29763b0f79d.png)
+
+
+# Procedure
+## Steps
+1. Participants read consent form and agree to participate
+2. Participants are given written instructions of the task 
+3. Participants answer a practice/example problem. 
+4. Participants then complete 36 RME picture problems.  
+> In the RME test, participants are shown 36 pairs of eyes, where for each pair of eyes they had to choose one of four words describing the corresponding emotion 
+5. Participants then complete a short survey 
```

Comment:
Do we have any information on questions asked in the survey?  I did not se anything in the material.

Reply:
Also there is no mention of the original test, I looked through the article referenced, but I think the information you have provided is solid.